### PR TITLE
[Event] Drop `organization_identifier` column part 1/3

### PR DESCRIPTION
**Pull request by rluodev**
This part: removes usages of column and ignores it, and also creates migration to set column to nullable so we don't have issues with creating new `event`s.

Next part: drops the column

Third part: unignores column

### CODE REVIEW

File: db/schema.rb
Comment (sampoder) [inline db/schema.rb:1]: Check this for unrelated changes!

Comment (rluodev) [reply db/schema.rb:1]: fixed!

**Review by garyhtou — APPROVED** [review]
